__tests__/project.test.js

```diff
@@ -39,3 +47,16 @@ describe("When passed a none existen path", () => {
       });
   });
 });
+
+describe("GET/api", () => {
+  test("200 returns all the documentation from the documentation file", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveProperty("GET /api");
+        expect(body).toHaveProperty("GET /api/topics");
+        expect(body).toHaveProperty("GET /api/articles");
```

Comment:
feels like this test could be more thorough.  i'd require in endpoint json and compare the body to that